metabase/metabase
# Add missing type in the base64 string for cards

Closes https://github.com/metabase/metabase/issues/34517

### Description

Add missing type in the base64 url string for models when updating the url.

This fixes the broken state of the notebook editor when refreshing the page while building model. 

### How to verify


1. New -> Model
2. Select "Use the notebook editor"
3. On the editor page, reload the page
4. Check the url bar, it should be `/model/query#ey..`


[Stress test for the new e2e test](https://github.com/metabase/metabase/actions/runs/9935926282)
